rllib/agents/sac/sac_policy.py

```diff
@@ -249,6 +287,29 @@ def gradients(policy, optimizer, loss):
     return grads_and_vars
 
 
+def apply_gradients(policy, optimizer, grads_and_vars):
+    actor_apply_ops = policy._actor_optimizer.apply_gradients(
+        policy._actor_grads_and_vars,
+        global_step=tf.train.get_or_create_global_step())
```

Comment:
Does `apply_gradients` automatically increment the global step? If it does, then I think we should not have this both here and below.

Reply:
Fixed! Only one optimizer (alpha optimizer) will increment the global step.